components/onboarding_explore_tools_tour/boards_tour_tip.tsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 
 import {FormattedMessage} from 'react-intl';
 
-import {useMeasurePunchouts} from 'components/widgets/tour_tip';
```

Comment:
i think this is where selective imports holds value, for example it is not sure which component this hooks belongs to (apart from documentation)
```
import {useMeasurePunchouts} from '@mattermost/components/tour_tip"
```
cc @hmhealey @crspeller

Reply:
But hooks component association I don't think is appropriate here? That useMeasurePunchouts isn't specific to the tour tip components (it just happens to be the only place it's used)

Reply:
Yeah it can be used anywhere, I will move it to common hooks

Reply:
What is `useMeasurePunchouts` actually intended to do? I realize now that I can't understand where else it'd be used because I don't actually know what it does to begin with 😛

And in general, I don't think we should be moving things to be common utilities until we know we're actually going to use them in multiple places. I think that one of the reasons that the current web app utils are so bloated because there's a bunch of one-off functions that ended up in there

Reply:
It helps in finding the coordinates and width & height of an element by Id, I think it can have another use case for sure.
Right now we use those for creating a cutout in the background overlay to interact with elements behind the overlay.